.github/workflows/docker-build-push-backend-container-on-tag.yml

```diff
@@ -3,61 +3,61 @@ name: Build and Push Backend Image on Tag
 on:
   push:
     tags:
-      - '*'
+      - "*"
 
 env:
-  REGISTRY_IMAGE: danswer/danswer-backend
+  REGISTRY_IMAGE: ${{ contains(github.ref_name, 'cloud') && 'danswer/danswer-backend-cloud' || 'danswer/danswer-backend' }}
   LATEST_TAG: ${{ contains(github.ref_name, 'latest') }}
```

Comment:
is this going to tag cloud images as latest?

Reply:
Believe we do want this functionality within docker– the latest tag should apply separately to the docker repositories (ie open source users would never see the `latest` cloud tags, is my understanding)

Reply:
ok, completely separate docker images. sounds good!